src/nvim/eval.c

```diff
@@ -15140,8 +15140,11 @@ static void f_sockconnect(typval_T *argvars, typval_T *rettv, FunPtr fptr)
     return;
   }
 
+  snprintf((char *)IObuff, sizeof(IObuff), "%s:%" PRIdLINENR,
```

Comment:
Well the information we currently have is available in a (module?) global, it could be accessed through a helper function that can be improved later. I can show what I mean in the channels PR.